framework/packages/abstract-client/src/builder.rs

```diff
@@ -155,6 +155,13 @@ impl<Chain: CwEnv> AbstractClientBuilder<Chain> {
     }
 }
 
+impl<A: cosmwasm_std::Api, S: StateInterface> AbstractClientBuilder<MockBase<A, S>> {
+    /// Deploy mock abstract with current configuration
+    pub fn build_test(&self) -> AbstractClientResult<AbstractClient<MockBase<A, S>>> {
```

Comment:
I prefer `build_mock`, wdyt?
```suggestion
    pub fn build_mock(&self) -> AbstractClientResult<AbstractClient<MockBase<A, S>>> {
```

Reply:
I assume we will have such helper for clone testing env as well. Would you prefer to call it `build_clone()`?

Reply:
Which one? The helper for clone-testing? 

Yeah, but maybe there it would be `build_clone_env` otherwise the "clone" is a bit ambiguous.